manifests/2.3.0/opensearch-2.3.0.yml

```diff
@@ -25,3 +25,9 @@ components:
   - name: security
     repository: https://github.com/opensearch-project/security.git
     ref: 2.x
+  - name: performance-analyzer
+    repository: https://github.com/opensearch-project/performance-analyzer.git
+    ref: 2.2
```

Comment:
Hi @kaushalmahi12, the reason python test failed is because you would need single quote for this floating type of reference. 
Also since you are modifying on the 2.3 manifest, this shouldn't be on 2.2 branch as currently i see its version is 2.2.1. 
For this PR, this is designated to be the main branch as the version bump [here](https://github.com/opensearch-project/performance-analyzer/pull/258) on the main branch.

Reply:
Ideally I think Performance Analyzer should create a `2.x` branch from their current `main` and specify that here in the 2.3.0 manifest. This is consistent with the branching strategy agreed upon for OpenSearch and it'll avoid having to change this again when their `main` inevitably upgrades to `3.0`

Reply:
Hi folk, I will point to 2.3 branch in that case.